### ISSUE TITLE
HK, ROSEN, Hernia 510: install Criteria Checker as per Amanda's indications

### ISSUE THREAD

**Issue by jesusitd** [intakedesk/PowerBI-General#478]
Launched `Hernia 510-One Criteria tab`:
![image](https://user-images.githubusercontent.com/50643267/75611921-c315f800-5aec-11ea-91be-ec90877bdc1a.png)

**Comment by jesusitd** [comment]
Waiting on comments. Deployed in production.

**Comment by jesusitd** [comment]
Duplicate of #483
